src/eos-updater-poll-common.c

```diff
@@ -652,12 +659,30 @@ eos_update_info_to_string (EosUpdateInfo *update)
   timestamp = eos_update_info_get_commit_timestamp (update);
   timestamp_str = g_date_time_format (timestamp, "%FT%T%:z");
 
-  return g_strdup_printf ("%s, %s, %s, %s\n   %s",
+  results_string = g_string_new ("");
+  for (i = 0; update->results != NULL && update->results[i] != NULL; i++)
+    {
+      const OstreeRepoFinderResult *result = update->results[i];
+
+      if (i != 0)
+        g_string_append (results_string, "\n   ");
```

Comment:
If we want all the lines of this to be indented, then the whitespace padding should probably go on the `g_string_append_printf` line below.

Reply:
The initial indentation is part of the final `g_strdup_printf()` call (with the `return`), below. It would be simpler to refactor as you suggest though.

Reply:
Fixup pushed.